### DISCUSSION TITLE
Problem with being recognized by dragcave.net

### DISCUSSION THREAD

**Discussion by Peshij** [curl/curl#12001] [Q&A]
Hi, I'm very new to curl usage, please bear with me.
I have troubles with requests made by curl being recognized by user pages of dragcave.net
Curl doesn't seem to be picking up cookies. Normally refreshing user page in browser increases number of views in Views column.
This is example of user page [https://dragcave.net/user/Penarrow101](url)
I've created batch file on Windows to run requests in loop.
`for /l %%x in (1, 1, 10) do (
curl.exe https://dragcave.net/user/Penarrow101 --user-agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 OPR/102.0.0.0" -b newcookies.txt https://dragcave.net/user/Penarrow101 --cookie-jar newcookies.txt https://dragcave.net/user/Penarrow101 --verbose
)
pause`
But Views do not increase according to number of iterations, it seems site does not recognize requests as valid.
Am I making some silly mistake somewhere?

**Comment by dfandrich** [comment]
The site has probably taken measures to prevent script kiddies from artificially inflating their view counts using a curl script.

**Reply by Peshij** [reply]
Might be true, digging in site sources would be required. Script kiddie has no problem with using autorefresher extension for Chrome which does the same, but is curious why curl doesn't work like Chrome.
